lib/api/open-test-in-editor.js

```diff
@@ -7,6 +7,6 @@ module.exports = async (req, res) => {
     await openInEditor(file);
     res.json('OK');
   } catch (e) {
-    res.status(422).send({ message: e.stderr.toString().trim() });
+    res.status(422).send({ message: 'Editor can\'t be started' });
```

Comment:
could you send, both error message & error description???

Reply:
ok